homeassistant/components/rflink/__init__.py

```diff
@@ -265,6 +275,7 @@ async def connect():
         _LOGGER.info("Connected to Rflink")
 
     hass.async_create_task(connect())
+    async_dispatcher_connect(hass, SIGNAL_EVENT, event_callback)
```

Comment:
Why do you need to handle the `SIGNAL_EVENT` also inside RFLink, does it allow sending a raw command without addressing the underlying entities?